projects/packages/my-jetpack/_inc/components/plans-section/index.jsx

```diff
@@ -76,23 +75,44 @@ function PlanSectionFooter( { purchases } ) {
 
 	const { recordEvent } = useAnalytics();
 
-	const clickHandler = useCallback( () => {
+	const purchaseClickHandler = useCallback( () => {
+		const event = purchases.length
+			? 'jetpack_myjetpack_plans_manage_click'
+			: 'jetpack_myjetpack_plans_purchase_click';
```

Comment:
pardon this question, does it make sense to define the event name depending on the number of purchases?

Reply:
For this event, yes, I think so. Because the link text also changes based on if there are purchases. But, for the license activation link, probably not. I'll fix that below.